.github/workflows/sdk-demo-beta-release.yaml

```diff
@@ -0,0 +1,35 @@
+name: Publish SDK demo on beta/stage release
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'releases/sdk/**-beta'
+
+jobs:
+  push_sdk_demo_beta:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
+      - name: Build SDK Demo
+        run:  npx lerna run build --scope @meeco/sdk-demo
+      - name: Deploy SDK Demo GH page
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          deploy_key: ${{ secrets.SDK_DOCS_DEPLOY_KEY }}
+          publish_dir: ./docs/sdk-demo
+          destination_dir: stage/sdk-demo
```

Comment:
Should this be `sdk-demo-beta` or some other subfolder so it doesn't overwrite the `master` demo?

Reply:
confusing huh? publish_dir = directory to publish from!

Reply:
Ah - I missed the `stage/` subdirectory - my bad 😅